### PR TITLE
Update mainnet-erc1155.md

### PR DESCRIPTION
**Pull request by Crypt0P0rtf0li0**

### CODE REVIEW

File: help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md [hunk 1]
```diff
@@ -87,3 +87,121 @@ Shoutout to CryptoPortfolio for creating this script.
 ```jsx title="ERC1155 TokenIds Script"
 
 ```
+
+This Node.js script allows you to fetch data from the OpenSea API for a specific collection and save the token IDs to a text file. It handles rate limiting and pagination automatically.
+
+## Prerequisites
```
Comment (iSpeakNerd) [inline help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md:93]: make level 3 header
`### `

File: help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md [hunk 2]
```diff
@@ -87,3 +87,121 @@ Shoutout to CryptoPortfolio for creating this script.
...
+
+## Prerequisites
+
+Before using this script, ensure you have the following prerequisites:
+
+1. Node.js installed on your system. (https://nodejs.org/en/download/)
+2. An OpenSea collection name. (Tip: Copy colection name directly from OpenSea Url)
+3. An API key from OpenSea. (https://docs.opensea.io/reference/api-keys)
+
+## Instructions
```
Comment (iSpeakNerd) [inline help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md:101]: make level 3 header 
`###`

File: help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md [hunk 3]
```diff
@@ -87,3 +87,121 @@ Shoutout to CryptoPortfolio for creating this script.
...
+
+- The script includes a rate-limiting mechanism to handle HTTP 429 (rate limit exceeded) responses. It will automatically retry the request after a delay (30-second) if a rate-limiting error occurs.
+
+- You can customize the delay between requests by modifying the `await delay(10000);` line (10 seconds in this example). Adjust it as needed to comply with OpenSea's rate limits.
+
+- If you encounter any issues or errors, review the console output for error messages and take appropriate action.
+
+- Make sure to keep your API key confidential and do not share it with others.
+
+```js
```
Comment (iSpeakNerd) [inline help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md:138]: change to
```
jsx title="ERC1155 TokenIds Script"
```

File: help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md [hunk 4]
```diff
@@ -87,3 +87,121 @@ Shoutout to CryptoPortfolio for creating this script.
 ```jsx title="ERC1155 TokenIds Script"
```
Comment (iSpeakNerd) [inline help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md:87]: remove this code block

File: help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md [hunk 5]
```diff
@@ -87,3 +87,121 @@ Shoutout to CryptoPortfolio for creating this script.
 ```jsx title="ERC1155 TokenIds Script"
 
 ```
+
```
Comment (iSpeakNerd) [inline help-docs/command-center/create-a-tgr/evm/mainnet-erc1155.md:90]: Add level 2 header
```
## Get All ERC1155 tokenIds Via OpenSea API
```

**Review by iSpeakNerd — CHANGES_REQUESTED** [review]
check the comments and resolve please @Crypt0P0rtf0li0

**Comment by Crypt0P0rtf0li0** [comment]
Changes updated.

**Review by iSpeakNerd — APPROVED** [review]
looks good ty

**Pull request merged**